pilot/pkg/serviceregistry/kube/controller/controller.go

```diff
@@ -267,7 +266,7 @@ type Controller struct {
 	// index over workload instances from workload entries
 	workloadInstancesIndex workloadinstances.Index
 
-	multinetwork
```

Comment:
nit: May be we should name it netwokManager. Usages of controller.network.network is hard to read and understand